EnglishTranslation-master/ITEM.tsv

```diff
@@ -9818,18 +9818,18 @@ ITEM_20151224_009817	Spell Amplifier
 ITEM_20151224_009818	An amplifier carried by Black Maizes. An item wanted by Revelator Mihail.
 ITEM_20151224_009819	Raffly Soul Crystal
 ITEM_20151224_009820	Vubbe Warrior Mind Crystal
-ITEM_20160224_009821	Many followed suit once one of the heirs of the more prestigious families fashioned their hair this way. It must be said however, that it is difficult to diverge all of its charm without the person being attractive in the first place.
-ITEM_20160224_009822	A hairstyle restored around the time of the Millenary which many people tried to imitate. However, it was mostly done by the nobility because it is difficult to cut and maintain.
+ITEM_20160224_009821	Once the heir of one the more prestigious families fashioned their hair this way, many followed suit. It must be said however, it's difficult to capitalize on its charm without being attractive in the first place.
+ITEM_20160224_009822	A hairstyle imitated by many, restored from around the time of the Millenary. However, it was mostly done by nobility due to it being difficult to cut and maintain.
 ITEM_20160224_009823	Root Lifted Wave Hairstyle
-ITEM_20160224_009824	Some say that a mere hairstyle cannot become the starting point of showing one's respect for the goddesses.{nl}However, goddesses also have preferences.
+ITEM_20160224_009824	Some say that a mere hairstyle cannot be a way to show one's respect for the goddesses.{nl}However, goddesses also have preferences.
 ITEM_20160224_009825	Noble Ponytail
-ITEM_20160224_009826	A hairstyle that has been in fashion each time the kingdom was at peace. This hairstyle signifies the longing for those peaceful times in these troublesome times after Medzio Diena.
-ITEM_20160224_009827	According to some rumors, General Ramin was able to find the princess in the ruins of the old capital when she was half buried in debris thanks to this hairstyle.
+ITEM_20160224_009826	A hairstyle that's came into fashion each time the kingdom was at peace. This hairstyle signifies longing for the peace of times past, before Medzio Diena.
+ITEM_20160224_009827	According to some rumors, General Ramin was able to find the princess half buried in the ruins of the old capital thanks to this hairstyle.
 ITEM_20160224_009828	This hairstyle has an inornate natural beauty about it. Much like the blessings of the goddesses.
 ITEM_20160224_009829	Noble Long
 ITEM_20160224_009830	Originally a hairstyle for esteemed daughters attending the Royal Girl's School in the old capital, it has since been imitated by many women.
 ITEM_20160224_009831	Short Cut Firm
-ITEM_20160224_009832	This expression that best fits this hairstyle is that it seems as if the sanctity of the Goddess Statue has descended upon the hair. Not the wooden statue of course.
+ITEM_20160224_009832	A metaphor that best fits this hairstyle is that it's as if the sanctity of the Goddess Statue has descended upon one's hair. Not the wooden statue of course.
```

Comment:
It's a square / rectangle kind of thing.  Not all metaphors are similes, but all similes are metaphors.  I feel like outside the world of literature, using metaphor is more appropriate.